<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NavForm</class>
 <widget class="QWidget" name="NavForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>302</width>
    <height>587</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>302</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>348</width>
       <height>100</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">

color: rgb(105, 137, 158);</string>
     </property>
     <property name="text">
      <string>NetAnalysisAssistant</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="toolButton_card">
     <property name="minimumSize">
      <size>
       <width>284</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Network Card</string>
     </property>
     <property name="icon">
      <iconset resource="uiFile.qrc">
       <normaloff>:/image/Cards.png</normaloff>:/image/Cards.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>40</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="toolButton_tcp">
     <property name="minimumSize">
      <size>
       <width>284</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Tcp Package</string>
     </property>
     <property name="icon">
      <iconset resource="uiFile.qrc">
       <normaloff>:/image/TCP.png</normaloff>:/image/TCP.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>80</width>
       <height>60</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="toolButton_udp">
     <property name="minimumSize">
      <size>
       <width>284</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>real Package</string>
     </property>
     <property name="icon">
      <iconset resource="uiFile.qrc">
       <normaloff>:/image/Tool.png</normaloff>:/image/Tool.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>80</width>
       <height>60</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="toolButton_capture">
     <property name="minimumSize">
      <size>
       <width>284</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Capture</string>
     </property>
     <property name="icon">
      <iconset resource="uiFile.qrc">
       <normaloff>:/image/Grap.png</normaloff>:/image/Grap.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>80</width>
       <height>60</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="toolButton">
     <property name="minimumSize">
      <size>
       <width>284</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
     <property name="icon">
      <iconset resource="uiFile.qrc">
       <normaloff>:/image/stop.png</normaloff>:/image/stop.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>80</width>
       <height>60</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>267</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="uiFile.qrc"/>
 </resources>
 <connections/>
</ui>
